client/app/dedicated/server/bandwidth/bandwitdh.module.js

```diff
@@ -0,0 +1,20 @@
+import publicOrder from './public-order/public-order.component';
+import publicCancel from './public-cancel/public-cancel.component';
+import privateOrder from './private-order/private-order.component';
+import privateCancel from './private-cancel/private-cancel.component';
+
+import publicOrderRouting from './public-order/public-order.routing';
+import publicCancelRouting from './public-cancel/public-cancel.routing';
+import privateOrderRouting from './private-order/private-order.routing';
+import privateCancelRouting from './private-cancel/private-cancel.routing';
+
+angular
```

Comment:
I feel like it would be better if you declared components with their associated routing and that's it :)

Reply:
I think you mean modules. I'll take a look

Reply:
Ah yes that's the word 😆

Reply:
Done